Src/BootCamp.Chapter/EventManager.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public class EventManager
+    {
+        public static Events ManageEvents()
```

Comment:
It's a bit odd to see this. Semantically it's correct, but logically it feels wrong. You should have events attached to an object that invokes them. Non-static object.

Reply:
Like in lessons example, I attached the events not to some events class, but to the objects that actually call them- like monster who can get hit or die. 

I think for you it makes more sense to have a Demo class to have those events.

Reply:
oke, so the Events class needs to be renamed to Demo or the EventManager class ?

Reply:
Get rid of the events class, it just adds not needed complexity. Move the events where they logically belong, close (or inside) the object that should invoke them.

Reply:
oke, then I have to think well where they belong

Reply:
so if you can explain what you mean. I still do not see it. As far as I see it I did everything the same as the examples you used for the lessons.

Reply:
also solved on the chat